cypress/e2e/pages/mediaAssetPage/index.cy.js

```diff
@@ -1,11 +1,353 @@
+    service: 'zhongwen',
+    runforEnv: ['live'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/simp/uk-23283128', // CPS Audio
+    service: 'zhongwen',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/simp/multimedia/2016/11/161107_tc2_testmap1?renderer_env=test', // TC2 Video
+    service: 'zhongwen',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/trad/world-53107744', // CPS video
+    service: 'zhongwen',
+    runforEnv: ['live'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/trad/uk-23283128', // CPS Audio
+    service: 'zhongwen',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/trad/multimedia/2016/11/161107_tc2_testmap1?renderer_env=test', // TC2 Video
+    service: 'zhongwen',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+];
+
+// non Amp Non Smoke Test Suites
+const nonAmpOnlyNonSmokeTestSuites = Cypress.env('SMOKE')
+  ? []
+  : [
+      {
+        path: '/afrique/institutionelles/2015/07/150714_hissene_habre_explainer', // TC2 MAP
+        service: 'afrique',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+      {
+        path: '/arabic/multimedia/2016/06/160601_qatar_sewika_smoking', // TC2 video
+        service: 'arabic',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+      {
+        path: '/hindi/multimedia/2015/09/150921_what_is_innovation_ms', // TC2 video
+        service: 'hindi',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+      {
+        path: '/russian/multimedia/2012/04/120411_v_titanic_last_survivor', // TC2 video
+        service: 'russian',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+      {
+        path: '/swahili/medianuai/2016/05/160517_apatae_fatacky', // TC2 MAP with video clip
+        service: 'swahili',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+      {
+        path: '/urdu/multimedia/2014/11/141104_hindu_riaz_kq', // TC2 MAP with video clip
+        service: 'urdu',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+      {
+        path: '/zhongwen/simp/multimedia/2016/06/160608_vid_gaokao_voxpop', // TC2 video
+        service: 'zhongwen',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+      {
+        path: '/zhongwen/trad/multimedia/2016/06/160608_vid_gaokao_voxpop', // TC2 video
+        service: 'zhongwen',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+    ];
 
-runTestsForPage(testsForPage);
+const canonicalTestSuites = Cypress.env('SMOKE')
+  ? canonicalSmokeTestSuites
+  : canonicalNonSmokeTestSuites;
+
+const ampTestSuites = canonicalTestSuites.map(testSuite => {
+  return {
+    ...testSuite,
+    path: `${testSuite.path}.amp`,
+    tests: [...ampTests],
+  };
+});
+
+// const liteTestSuites = Cypress.env('SMOKE')
```

Comment:
Shall we try to include these?

Reply:
Ignore this, I've just looked at the tests, and they only check for the lite site summary link, which is on all lite pages, so we should be OK to reuse. I think we should extract these tests to a shared location, as they are now used in multiple tests